StringCatalogEnum/Sources/StringCatalogEnum/main.swift

```diff
@@ -36,13 +36,17 @@ struct StringCatalogEnum: ParsableCommand {
         let data = try Data(contentsOf: url)
         print(data)
 
-        guard let json = try JSONSerialization.jsonObject(with: data, options: []) as? [String: Any] else {
-            throw Error.unexpectedJSON(message: "cannot parse first level object")
-        }
+        // guard let json = try JSONSerialization.jsonObject(with: data, options: []) as? [String: Any] else {
+        //     throw Error.unexpectedJSON(message: "cannot parse first level object")
+        // }
+
+        // guard let strings = json["strings"] as? [String: Any] else {
+        //     throw Error.unexpectedJSON(message: "cannot parse `strings`")
+        // }
```

Comment:
As a general, don't include dead code in your PRs. If you really have a good reason, add a `TODO` or something to explain why the dead code would be needed.